server/data/data-api/src/main/java/org/apache/james/rrt/api/RecipientRewriteTable.java

```diff
@@ -122,6 +122,8 @@ default Mappings getResolvedMappings(String user, Domain domain) throws ErrorMap
      */
     Map<MappingSource, Mappings> getAllMappings() throws RecipientRewriteTableException;
 
+    int getMappingLimit();
+
     default Stream<MappingSource> listSources(Mapping mapping) throws RecipientRewriteTableException {
```

Comment:
Should be moved to `ReverseRecipientRewriteTable` ?

Reply:
I changed the semantic